python/cugraph/linear_assignment/lap.pxd

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cugraph.structure.graph_primtypes cimport *
+
+cdef extern from "algorithms.hpp" namespace "cugraph":
+
+    cdef void hungarian[VT,ET,WT](
+        const handle_t &handle,
+        const GraphCOOView[VT,ET,WT] &graph,
```

Comment:
Should we be standardizing on the new "graph container" pattern for new algos (ie. this would accept a `const graph_container_t& g` instead)?

Reply:
I wrote much of this before the graph container paradigm was created.  I could adapt this if we think it's important to get in now or we can put an issue on the backlog to fix this later.

Reply:
backlog makes more sense if we're shooting for 0.17 IMO.